packages/react/src/generators/component-story/component-story.spec.ts

```diff
@@ -217,6 +217,67 @@ describe('react:component-story', () => {
       });
     });
 
+    describe('aaacomponent with props and actions', () => {
+      beforeEach(async () => {
+        appTree.write(
+          cmpPath,
+          `import React from 'react';
+  
+          import './test.scss';
+
+          export type ButtonStyle = 'default' | 'primary' | 'warning';
+          
+          export interface TestProps {
+            name: string;
+            displayAge: boolean;
+            someAction: (e: unknown) => void;
+            style: ButtonStyle;
```

Comment:
Just pushed this change to the test which results in a failure. I think we should only add props to the story for which we're able to grab the default value. Otherwise we should rely on Storybook's controls autogeneration mechanism.

Oh...also plz remove the `aaa` in the `describe('aaacomponent`. Accidentally left that in my commit

Reply:
Ohh nice catch!!